README.md

```diff
@@ -0,0 +1,30 @@
+# Hackers at Cambridge Website
+
+This website is written using the Swift web framework [Kitura](https://github.com/IBM-Swift/Kitura) to allow us the flexibility to do what we please with the website and to allow us to try out Swift as an option for server-side programming.
+
+## Installing and running the website
+This project uses the [Gulp task runner](http://gulpjs.com) to make this super simple. 
+
+First make sure you have [Node](https://nodejs.org/en/) (you will need `npm` to install and run Gulp) and [Swift 3](https://swift.org/download/) installed 
+
+Clone this repository
+
+```
+git clone https://github.com/hackersatcambridge/hac-website.git
+```
+
+And then run:
+
+```
+npm install
```

Comment:
No reason no. There is nothing specific to Swift on the JS side so we can do whatever we like. Happy to use Yarn